### DISCUSSION TITLE
GITHUB_COM_TOKEN missing after upgrade from 29.14.0 to  31.11.2

### DISCUSSION THREAD

**Discussion by volker-raschek** [renovatebot/renovate#13222] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

31.11.2

### Please select which platform you are using if self-hosting.

Gitea

### If you're self-hosting Renovate, tell us what version of the platform you run.

1.15.2

### Describe the bug

Failed to start renovate after update from 29.14.0-slim to 31.11.2. 

```txt
FATAL: You must configure a GitHub personal access token
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 60,
           "logContext": "3qLyZ7HODyaf_9Cu_SmYA",
           "msg": "You must configure a GitHub personal access token"
         }
       ]
```

I specified the github.com token via the environment variable `GITHUB_COM_TOKEN` as described [here](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/running.md#githubcom-token-for-release-notes), but I receive the error. 

Maybe the environment variable has been changed or is something else miss configured? To be honest, I don't know what to do. I couldn't find anything more about the token for github.com. Here is my config, if it helps.

```json
    {
      "autodiscover": true,
      "endpoint": "<marked>",
      "logFile": "/dev/stdout",
      "logFileLevel": "debug",
      "platform": "<marked>",
      "token": "<marked>"
    }
```

Volker

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
This is not because of `GITHUB_COM_TOKEN`. The message you received will only be shown if Renovate thinks you are running on a github code platform. You need to ensure `platform=gitea`.

Run with `LOG_LEVEL=debug` in env and you can maybe see from that where things went wrong.

**Comment by volker-raschek** [comment]
Hi @rarkins 

I started renovate again with `LOG_LEVEL=debug`. Here are the logs: 

```txt 
DEBUG: Using RE2 as regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in /usr/src/app/config.js
DEBUG: No config file found on disk - skipping
DEBUG: Converting GITHUB_COM_TOKEN into a global host rule
DEBUG: File config
       "config": {}
DEBUG: CLI config
       "config": {}
DEBUG: Env config
       "config": {
         "hostRules": [
           {"hostType": "github", "matchHost": "github.com", "token": "***********"}
         ]
       }
DEBUG: Combined config
       "config": {
         "hostRules": [
           {"hostType": "github", "matchHost": "github.com", "token": "***********"}
         ]
       }
DEBUG: Found valid git version: 2.34.1
FATAL: You must configure a GitHub personal access token
DEBUG: Renovate exiting
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 60,
           "logContext": "QyY-tnTaebB05qkzcL85P",
           "msg": "You must configure a GitHub personal access token"
         }
       ]
```

The GITHUB_COM_TOKEN seems to be parsed, even if the error message is wrong.

I'm surprised that in the new version it doesn't find `config.json` and only looks for `config.js`. After all, a `config.json` is mounted in the into the [kubernetes pod](https://github.com/volker-raschek/renovate-charts/blob/master/templates/cronjob.yaml#L56).

**Reply by rarkins** [reply]
You should be able to pass `RENOVATE_CONFIG_FILE=/usr/src/app/config.json` although I had thought it should be automatically tried if the `config.js` is not found.

@viceice I thought we'd agreed to try for `config.json` too..

**Reply by viceice** [reply]
No, we skipped that check, because of complexity

**Reply by rarkins** [reply]
OK thanks, I updated https://github.com/renovatebot/renovate/releases/tag/30.0.0 for clarity
